### ISSUE TITLE
Training loop

### ISSUE THREAD

**Issue by laudominik** [Gradient-PG/gradient-tts#4]
Training loop is supposed to:
- use 3 second acoustic prompts
- use between 10 to 20 second waveforms
- loss: crossentropy
- linear decay
- AdamW optimizer
